.github/workflows/release.yml

```diff
@@ -132,46 +130,46 @@ jobs:
         if: matrix.platform == 'win32'
         run: |
           cd target/${{ matrix.rust-target }}/ci
-          cp typstyle.pdb typstyle-${{ env.target }}.pdb
+          cp typstyle.pdb typstyle-${{ matrix.rust-target }}.pdb

```

Comment:
what is rust-target? is it defined by `dtolnay/rust-toolchain` ?

Reply:
`rust-target` is defined above (`strategy.matrix`)